packages/admin-next/dashboard/package.json

```diff
@@ -33,6 +33,7 @@
     "@dnd-kit/sortable": "^8.0.0",
     "@hookform/resolvers": "3.3.2",
     "@medusajs/icons": "1.2.1",
+    "@medusajs/js-sdk": "0.0.2-snapshot-20240516152256",
```

Comment:
You should just add the version that is in the sdk's package.json. Then changeset will take care of updating it with each release, and yarn will point it towards the local dependency.